src/compiler.adoc

```diff
@@ -1067,7 +1067,8 @@ of the `core_tests.cljs` file:
 (set! *main-cli-fn* #(t/run-tests))
 ----
 
-You can again run the compiled file to check that the new test passes as expected.
+Run the compiled file again to see that there are now two tests running.  The
+first test fails as it did before, but our leap year test passes as expected.
```

Comment:
clarify: mention that the first test still fails, in case reader is confused to not see full success